lib/lint.js

```diff
@@ -15,6 +16,7 @@ const STANDARD_APP_PATHS = [
   'test-support',
   'tests',
 ];
+const STANDARD_JS_PATHS = [...STANDARD_APP_PATHS, 'blueprints'];
```

Comment:
do you think we can just drop `blueprints` from the list?

Reply:
Including it for JS really just lints `blueprints/my-blueprint/index.js`. The rest of the files are ignored by `.eslintignore` in the app.